recipe/meta.yaml

```diff
@@ -29,7 +29,7 @@ requirements:
     - python
     - pip
     - setuptools
-    - pybind11 >=2.5,<2.6
+    - pybind11 >=2.5
```

Comment:
You could try setting this to a know good version of pybind11

Reply:
... and add a comment `pybind11 ==2.5  # pypy fails to build with 2.8

Reply:
@h-vetinari can you add me as collaborator to this project? Seems I dont have the permission to push changes to this branch. Thanks!

Reply:
There were no pypy builds for 2.5, AFAIR. There's definitely no cpython 3.10 build for pybind 2.5, so I'd be reluctant to go back (though we can, selectively, for pypy, if necessary)

Reply:
ah okay, then please ignore my suggestion 😢

Reply:
Not sure how this all works. If there is no conda package, is there a way to pip install? If I recall correctly pybind11 does not build any c-extensions, so the install should be cheap.

Reply:
no no, we'll try. 🙃 

And if it's really a specific pybind11-version that's bad for pypy, we could also rebuild the (missing) old pybind versions for pypy on the respective feedstock.